src/app/views/file_view/import_view.tsx

```diff
@@ -119,6 +119,9 @@ export class FileViewImport extends ContextedComponent<
                           <th className="charticulator__file-view-mapping_row_item">
                             Required data type
                           </th>
+                          <th className="charticulator__file-view-mapping_row_item">
+                            Examples
```

Comment:
In the last PR you started to extract string constants to strings.ts. Will we apply the same approach for all new/updated components?

Reply:
I'd like to get this PR in quickly to unblock this feature, I think it will be easier to do a language pass separately.